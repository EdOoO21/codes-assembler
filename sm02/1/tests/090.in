461 -811 -15 350 176 200 -366 299 815 -274 648 -76 180 819 985 675 -372 -458 0 -442 -243 -848 119 498 -706 -916
