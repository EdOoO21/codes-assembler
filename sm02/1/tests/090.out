-916
-706
498
119
-848
-243
-442
0
-458
-372
675
985
819
180
-76
648
-274
815
299
-366
200
176
350
-15
-811
461
